examples/screenshare/main.go

```diff
@@ -52,7 +52,7 @@ func main() {
 	s, err := md.GetDisplayMedia(mediadevices.MediaStreamConstraints{
 		Video: func(c *mediadevices.MediaTrackConstraints) {
 			c.CodecName = videoCodecName
-			c.FrameFormat = frame.FormatI444
+			c.FrameFormat = frame.FormatI420
```

Comment:
This prop is sent to the screen capture driver and just ignored.

Reply:
Ah. I'll remove it to make it clear. Also, I'll add a new const, `FormatRGBA` to correctly label the output from `Properties`